### DISCUSSION TITLE
MEMPOOL CONFLICT when attempting to submit a previously deleted transaction.

### DISCUSSION THREAD

**Discussion by Jsewill** [Chia-Network/chia-blockchain#9055] [Support]
I recently submitted a transaction, which was in showing a status of "in mempool", however, it went so long without being confirmed that I thought I'd run `chia wallet delete_unconfirmed_transactions` and try again later.

When I did try again later, I got something like, `Transaction submitted to nodes: [('<alphanumeric_string>', 2, 'MEMPOOL_CONFLICT')]`

What can I do to actually submit the transaction successfully?

**Comment by andrii-stasiuk** [comment]
i have submitted a bug https://github.com/Chia-Network/chia-blockchain/issues/9060

**Comment by Jsewill** [comment]
It appears that the first transaction went through. I do think this case needs additional handling, possibly returning an error if the transaction is too far along to be deleted.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
